stdlib/database/h2/build.gradle

```diff
@@ -4,11 +4,16 @@ dependencies {
     baloImplementation project(path: ':ballerina-builtin', configuration: 'baloImplementation')
     baloImplementation project(path: ':ballerina-sql', configuration: 'baloImplementation')
 
+    // transitive
+    baloImplementation project(path: ':ballerina-sql', configuration: 'baloImplementation')
+    baloImplementation project(path: ':ballerina-config-api', configuration: 'baloImplementation')
+    baloImplementation project(path: ':ballerina-system', configuration: 'baloImplementation')
```

Comment:
ballerina-system is not an implementation, is that correct. ?

Reply:
should be a testCompile dependency. Ill revisit this and reopen PR